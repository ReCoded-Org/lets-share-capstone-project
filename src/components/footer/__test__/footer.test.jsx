import renderer from "react-test-renderer";
import Footer from "../footer";

it("renders correctly", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
});
