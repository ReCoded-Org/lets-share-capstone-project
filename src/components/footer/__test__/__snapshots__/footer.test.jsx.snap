// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=" flex w-full flex-col items-center justify-around bg-primary p-5 text-2xl text-[white] md:flex-row "
  id="footer"
>
  <div
    className="cursor-pointer "
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "inline-block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "relative",
          "width": "initial",
        }
      }
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27100%27%20height=%27100%27/%3e"
          style={
            Object {
              "background": "none",
              "border": 0,
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        />
      </span>
      <img
        alt="Lets-share-logo"
        data-nimg="intrinsic"
        decoding="async"
        onClick={[Function]}
        onError={[Function]}
        onLoad={[Function]}
        onMouseEnter={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt="Lets-share-logo"
          data-nimg="intrinsic"
          decoding="async"
          loading="lazy"
          onClick={[Function]}
          onMouseEnter={[Function]}
          src="/_next/image?url=%2Fimages%2FLets-share-logo.png&w=256&q=75"
          srcSet="/_next/image?url=%2Fimages%2FLets-share-logo.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2FLets-share-logo.png&w=256&q=75 2x"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
  </div>
  <div
    className="p-5"
  >
    <ul>
      <li
        className="pb-2 hover:text-fontColor "
      >
        <a
          className=" "
          href="/aboutus"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          About us
        </a>
      </li>
      <li
        className="pb-2 hover:text-fontColor"
      >
        <a
          href="/blogs"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Blogs
        </a>
      </li>
      <li
        className="pb-2 hover:text-fontColor"
      >
        <a
          href="/goods"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Goods
        </a>
      </li>
    </ul>
  </div>
  <div
    className=" p-3 "
  >
    <ul
      className=""
    >
      <p
        className=" p-3 hover:text-fontColor "
      >
        Contact us
      </p>
      <div
        className="flex gap-3 text-3xl "
      >
        <svg
          className="cursor-pointer hover:text-fontColor"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-92.4 233.5h-63.9c-50.1 0-59.8 23.8-59.8 58.8v77.1h119.6l-15.6 120.7h-104V912H539.2V602.2H434.9V481.4h104.3v-89c0-103.3 63.1-159.6 155.3-159.6 44.2 0 82.1 3.3 93.2 4.8v107.9z"
          />
        </svg>
        <svg
          className="cursor-pointer hover:text-fontColor"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
          />
        </svg>
        <svg
          className="cursor-pointer hover:text-fontColor"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zM349.3 793.7H230.6V411.9h118.7v381.8zm-59.3-434a68.8 68.8 0 1 1 68.8-68.8c-.1 38-30.9 68.8-68.8 68.8zm503.7 434H675.1V608c0-44.3-.8-101.2-61.7-101.2-61.7 0-71.2 48.2-71.2 98v188.9H423.7V411.9h113.8v52.2h1.6c15.8-30 54.5-61.7 112.3-61.7 120.2 0 142.3 79.1 142.3 181.9v209.4z"
          />
        </svg>
        <svg
          className="cursor-pointer hover:text-fontColor"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11 2.5a2.5 2.5 0 1 1 .603 1.628l-6.718 3.12a2.499 2.499 0 0 1 0 1.504l6.718 3.12a2.5 2.5 0 1 1-.488.876l-6.718-3.12a2.5 2.5 0 1 1 0-3.256l6.718-3.12A2.5 2.5 0 0 1 11 2.5z"
          />
        </svg>
      </div>
    </ul>
  </div>
  <div
    className="p-5"
  >
    <ul>
      <p
        className=" pb-2"
      >
         Adress 
      </p>
      <p
        className=" pb-2"
      >
         Street 01,17 
      </p>
      <p
        className=" pb-2"
      >
         City, Country
      </p>
    </ul>
  </div>
</div>
`;
